Validation/RecoTau/python/RecoTauValidation_cff.py

```diff
@@ -19,25 +26,25 @@
 )
 tauValidationMiniAODRealData = tauValidationMiniAODZTT.clone(
   RefCollection = cms.InputTag("CleanedPFJets"),
```

Comment:
Please drop type, i.e.
tauValidationMiniAODZTT.clone(
  RefCollection = "CleanedPFJets",
ExtensionName = 'JETHT')
following point 17. on:
https://cms-sw.github.io/cms_coding_rules.html#6--packaging-rules-1

Reply:
Hello, these definitions are coming from already previously used releases and also in the CMSSW master branch [1]. Should we edit all the cloned types in this file ?
 
[1] https://github.com/cms-sw/cmssw/blob/master/Validation/RecoTau/python/RecoTauValidation_cff.py#L8-L31

Reply:
If it does not cost you much, yes please!

Reply:
Please bear in mind that these changes about drop type have been just implemented in this PR from the DQM central campaign:
https://github.com/cms-sw/cmssw/pull/36511